x-pack/plugins/upgrade_assistant/public/application/components/overview/deprecation_logging_toggle.tsx

```diff
@@ -13,8 +13,35 @@ import { i18n } from '@kbn/i18n';
 import { useAppContext } from '../../app_context';
 import { ResponseError } from '../../lib/api';
 
+const i18nTexts = {
+  toggleErrorLabel: i18n.translate(
+    'xpack.upgradeAssistant.overviewTab.steps.deprecationLogsStep.enableDeprecationLoggingToggleSwitch.errorLabel',
+    {
+      defaultMessage: 'Could not load logging state',
+    }
+  ),
+  toggleLabel: i18n.translate(
+    'xpack.upgradeAssistant.overviewTab.steps.deprecationLogsStep.enableDeprecationLoggingToggleSwitch.enabledLabel',
+    {
+      defaultMessage: 'Enable deprecation logging',
+    }
+  ),
+  enabledMessage: i18n.translate(
+    'xpack.upgradeAssistant.overviewTab.steps.deprecationLogsStep.enableDeprecationLoggingToggleSwitch.enabledToastMessage',
+    {
+      defaultMessage: 'Deprecation logs will be displayed in the Elasticsearch console.',
```

Comment:
👍 Yes, we have discussed either creating a UI specific for Upgrade Assistant to show the deprecation logs or directing users to Discover to view them. This will be a future enhancement. For now, I think updating the text to your suggestion makes sense.